appserver/payara-appserver-modules/microprofile/openapi/src/main/java/fish/payara/microprofile/openapi/impl/model/util/ModelUtils.java

```diff
@@ -427,6 +421,18 @@ public static boolean isAnnotationNull(Annotation annotation) {
         return allNull;
     }
 
+    public static Boolean mergeProperty(Boolean current, boolean offer, boolean override) {
+        return mergeProperty(current, Boolean.valueOf(offer), override);
+    }
+
+    public static Boolean mergeProperty(boolean current, Boolean offer, boolean override) {
+        return mergeProperty(Boolean.valueOf(current), offer, override);
+    }
+
+    public static Boolean mergeProperty(boolean current, boolean offer, boolean override) {
+        return mergeProperty(Boolean.valueOf(current), Boolean.valueOf(offer), override);
+    }
+
```

Comment:
FYI: I added this overloaded variants to avoid boxing/unboxing warnings. The module is almost free of warnings and it would be nice to get there.